test/runnable/extra-files/cppb.cpp

```diff
@@ -519,3 +519,35 @@ Visitor2* getVisitor2()
 {
     return &inst;
 }
+
+/******************************************/
+// issues detected by fuzzer
+#if _LP64
+#define longlong long
+#else
+#define longlong long long
```

Comment:
> Use int64_t...

Ok, done. The fuzzer could have written these to begin with ;-)
VC doesn't have stdint.h before VS2010, but that's probably no problem for the test suite.